pkg/api/frontendsettings.go

```diff
@@ -374,6 +380,15 @@ func isSupportBundlesEnabled(hs *HTTPServer) bool {
 	return hs.Cfg.SectionWithEnvOverrides("support_bundles").Key("enabled").MustBool(true)
 }
 
+// don't need to show the full commit hash in the UI
+// let's substring to 10 chars like local git does automatically
```

Comment:
I would remove this line of comment (the function doesn't specify the actual number of digits, it's up to the callsite).

Reply:
(Sorry, just saw when scrolling down that this piece of code was just moved around.)

Reply:
copy-pasting is not an excuse to not leave things better as we go along! I'll take a look at it :)